staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -184,6 +185,17 @@ func UpdateResource(r rest.Updater, scope *RequestScope, admit admission.Interfa
 			)
 			wasCreated = created
 			return obj, err
+		}
+		result, err := finishRequest(timeout, func() (runtime.Object, error) {
+			result, err := requestFunc()
+			if isTooLargeError(err) && scope.FieldManager != nil {
+				if accessor, accessorErr := meta.Accessor(obj); accessorErr == nil {
+					accessor.SetManagedFields(nil)
+					transformers = transformers[1:]
```

Comment:
this subslice is fragile... insertion at `[0]` above is conditional on `scope.FieldManager != nil`, and I could easily see the code above changing so that the field manager transformer didn't happen to be at index 0